docs/core/deploying/native-aot/index.md

```diff
@@ -143,10 +175,19 @@ Publishing a class library as native AOT creates a native library that exposes m
 
 The following table shows supported compilation targets.
 
+### [.NET 7](#tab/net7)
+
 | Platform | Supported architecture |
 |----------|------------------------|
 | Windows  | x64, Arm64             |
 | Linux    | x64, Arm64             |
-| macOS*   | x64, Arm64             |
 
-\* Supported starting in .NET 8.
+### [.NET 8+](#tab/net8plus)
+
+| Platform | Supported architecture |  Notes               |
+|----------|------------------------|----------------------|
+| Windows  | x64, Arm64             |                      |
+| Linux    | x64, Arm64             |                      |
+| macOS    | x64, Arm64             | Experimental support |
```

Comment:
```suggestion
| macOS   | x64, Arm64 |                                        |
| iOS     | x64, Arm64 | Experimental support                   |
| Android | x64, Arm64 | Experimental, no built-in Java interop |
```

@ivanpovazan what's the full matrix here? Should tvOS/watchOS/Catalyst also be called out (I don't actually know what we're bringing up :))?